test/integration/typescript/src/example.ts

```diff
@@ -14,12 +14,9 @@
  * limitations under the License.
  */
 
-// import { initializeApp, cert } from '../../../../lib';
-// import { FirebaseApp } from '../../../../lib/firebase-app';
-
 import * as firebase from 'firebase-admin';
```

Comment:
Not really relevant to this PR. Going forward we should use named imports in our own code too.